skimage/filters/_fft_based.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import scipy.fft as fft
+from skimage.exposure import rescale_intensity
+from skimage import dtype_limits
```

Comment:
```suggestion

from .._shared.fft import fft_module as fft
from ..exposure import rescale_intensity
from ..utils import dtype_limits
```

The change to the fft import is needed until we drop support for SciPy < 1.4 (it falls back to numpy.fft when scipy.fft is unavailable).

The others just changes the style to use relative imports for consistency with the rest of the library

Reply:
done, however I had to change it to `fftmodule` and `..util`